homeassistant/components/yamaha/media_player.py

```diff
@@ -193,6 +193,8 @@ class YamahaDevice(MediaPlayerEntity):
     def __init__(self, name, receiver, source_ignore, source_names, zone_names):
         """Initialize the Yamaha Receiver."""
         self.receiver = receiver
+        if name == DEFAULT_NAME and not receiver.model_name is None and not receiver.model_name == "Unknown":
+            name = 'Yamaha %s' % receiver.model_name
```

Comment:
We can't change the default name yet. The unique id needs to be released for at least one release, preferably more, first.